### DISCUSSION TITLE
Hiding/Disabling a single entry page in within a collection

### DISCUSSION THREAD

**Discussion by localghost8000** [statamic/cms#7709] [Q&A] [answered]
I have a collection of case studies. 99% of the time, these case studies are display on an index page, and have a corresponding entry page. However, in some (rare) cases, the client has requested that the case study only appear on the index page, and not have a corresponding entry page. 

Its easy enough to hide this from the user, but is there a simple method of ensuring that no page/slug for the entry is generated?

Appreciate any advice :)

**Comment by jasonvarga** [comment, marked as answer]
There's not really a way to prevent a single entry from having a URL.

But, you could have a `toggle` field called something like `prevent_url` and in your template trigger a 404.

```antlers
{{ if prevent_url }}{{ 404 }}{{ /if }}
```
